pyhelper_utils/general.py

```diff
@@ -67,14 +69,15 @@ def inner(*args: Any, **kwargs: Any) -> Any:
             try:
                 return func(*args, **kwargs)
             except Exception as ex:
-                if retry:
-                    sleep(retry_interval)
-                    for _ in range(0, retry):
-                        try:
-                            return func(*args, **kwargs)
-                        except Exception:
-                            sleep(retry_interval)
-
+                for idx in range(0, retry):
```

Comment:
start the range at 1, then you don;t need to do idx + 1 in line 77

Reply:
@myakove yes, the issue with that is when `retry=1` the for loop will not execute. I could add a `+1` to `retry` if you'd rather that.

Reply:
Leave it